gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Mar 24 11:47:35 IST 2023
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
```

Comment:
The grale build is done for the whole repo. Therefore no need to commit this file I think.

Reply:
Resolved through 1dc87e35f135e4ec5ba44a8440e3c1c565c0d329